docs/02-about-us/departments.md

```diff
@@ -0,0 +1,22 @@
+# Departments
```

Comment:
The primary benefit I see is that services roll up to departments, which have guilds. Services are intended to be client-facing so we can be really clear about what we offer people. Services can belong to a department that holds a routine governance meeting where "improvements" can be raised across any of the department's services.